pkg/client/chartmuseum/chartmuseumtester.go

```diff
@@ -138,12 +139,14 @@ func (rt *RepoTester) GetURL() string {
 func (rt *RepoTester) GetIndex(w http.ResponseWriter, r *http.Request, emptyIndex bool, indexFile string) {
 	w.Header().Set("Content-Type", "application/yaml")
 	w.WriteHeader(200)
+	_, filename, _, _ := runtime.Caller(1)
+	testdataPath := path.Join(path.Dir(filename), "../../../testdata")
```

Comment:
This "hack" is to avoid using a relative path to testdata folder as it depends on the test file invoking this method. Depeding on the test, this relative path was `../../testdata` or `../../../testdata`

Reply:
should we add a `testdata` folder for this package, specifically?

Reply:
not sure, I am reusing the same testdata for chartmuseum, harbor and helmclassic clients and also for the sync_test.go test